spec/features/admin/delete_custom_organisation_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'support/confirmation_use_case'
+
+describe 'deleting a custom organisation name' do
+  include_examples 'confirmation use case spy'
+  let!(:custom_org_name1) { CustomOrganisationName.create(name: 'DummyOrg1') }
+  let!(:custom_org_name2) { CustomOrganisationName.create(name: 'DummyOrg2') }
+  let!(:custom_org_name3) { CustomOrganisationName.create(name: 'DummyOrg3') }
+
+  let!(:admin_user) { create(:user, super_admin: true) }
+
+  context 'when visiting the custom organisations page' do
+    before do
+      sign_in_user admin_user
+      visit admin_custom_organisations_path
+    end
+
+    it 'allows the user to delete a custom organisation' do
+      html_selector = "tr#" + custom_org_name1.id.to_s
+      within(html_selector) do
+        click_link 'Remove'
+      end
```

Comment:
Haven't tested this, but would this work?

page.find('#organisation-#{custom_org_name1.id}'').click

Not sure we need to add ids to so many elements on that page

Reply:
Its so we can click the right remove button when deleting an organisation, so we need to make sure in the test that we delete the correct custom org